examples/tail_long_div.asm

```diff
@@ -8,15 +8,20 @@ div:
     ;; Slot 4: Return value quot
     ;; Slot 5: Return value rem
     ;; Slot 6: Non-deterministic local: Next FP
-    ;; Slot 7: 0x80000000
+    ;; Slot 7: Return quot value absolute addr
```

Comment:
Idea: maybe use this example to try setting multiple return values with a single pointer?